test/onnx/test_onnx_export.py

```diff
@@ -60,3 +61,35 @@ def break_is_registered_op_api(opname, domain, version):
                               operator_export_type=OperatorExportTypes.ONNX_ATEN_FALLBACK)
         onnx_model = onnx.load_from_string(f.getvalue())
         self.assertAtenOp(onnx_model, "clamp", "Tensor")
+
+
+    def _helper_test_to_(self, cast_fn: Callable[[torch.Tensor], torch.Tensor]):
+        """Run a generic test with the specified cast_fn
+
+        `cast_fn` will be converted into a `torch.jit.script` function
+        and it is meant to wrap a `aten::to` during export to prevent
+        hard-coded devices
+        """
+        cast_fn = torch.jit.script_if_tracing(cast_fn)
+
+        class MyModel(torch.nn.Module):
```

Comment:
do we even need a nn.Module? I think we can export a ScriptFunction directly.

Reply:
The original idea is to have a torch script wrapping device change on a `torch.nn.Module`, not a full torch script scenario

Reply:
I don't understand.
We're testing support for `aten:;to`. I think we can do that with just passing in `torch.onnx.export(torch.jit.script(cast_fn), ...)` without an `nn.Module` class. If we can test the functionality just as well that way, that seems better to me: fewer layers of things to understand, less lines of code to read and maintain.